scenarios/test_scenarios/gs_forced_collision_test.osm

```diff
@@ -1,28 +1,29 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <osm version='0.6' generator='JOSM'>
-  <node id='-5347261' action='modify' visible='true' lat='43.47161324274' lon='-80.53937569025'>
+  <node id='-5399555' action='modify' visible='true' lat='43.47161324274' lon='-80.53937569025'>
     <tag k='btype' v='PP' />
     <tag k='collision_vehicle_vid' v='1' />
     <tag k='cycles' v='1' />
     <tag k='gs' v='pedestrian' />
     <tag k='name' v='path_ped' />
     <tag k='path' v='pp1' />
     <tag k='pid' v='1' />
-    <tag k='speed' v='3' />
+    <tag k='speed' v='1' />
+    <tag k='speed_qualifier' v='minimum' />
```

Comment:
have you intended to use minimum here?

Reply:
The intent was that you change the speed_qualifier value each time to test the behavior. `minimum` was what it was set to at the time of pushing the scenario which I can change to something else if you want.

I did however push another test called `gs_collision_point_behind_ped.osm` specifically for the `minimum` speed qualifier.

Do you want me to have a test case for each speed qualifier? It will be the same or a similar scenario.

Reply:
Ideally, we'd have a test for each qualifier. But since you already have one for minimum, maybe change this one to maximum and that's it. We can merge.